packages/suricata.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,114 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+Import-Module powershell-yaml
+
+# set configurations
+$toolName = "Suricata"
+$category = VM-Get-Category($MyInvocation.MyCommand.Definition)
+$filetype = "MSI"
+$toolDir = Join-Path ${Env:ProgramFiles} $toolName
+$executablePath = Join-Path $toolDir "suricata.exe"
+$url = "https://www.openinfosecfoundation.org/download/windows/Suricata-7.0.8-1-64bit.msi"
```

Comment:
we call this `exeUrl` in other packages like [bindiff.vm](https://github.com/mandiant/VM-Packages/blob/af930c6eb0e13cffcca8c037e3469570785a4f80/packages/bindiff.vm/tools/chocolateyinstall.ps1#L7):
```suggestion
$exeUrl = "https://www.openinfosecfoundation.org/download/windows/Suricata-7.0.8-1-64bit.msi"
```